stdlib/public/core/Character.swift

```diff
@@ -211,22 +211,27 @@ extension String {
   }
 }
 
+@_needsUnicodeDataTablesInEmbedded
 extension Character: Equatable {
   @inlinable @inline(__always)
   @_effects(readonly)
+  @_needsUnicodeDataTablesInEmbedded
```

Comment:
In order for this to be intuitive for library authors, I think we should probably aim to support lexical scoping for this attribute so that you don't have to repeat it like this